openeogeotrellis/udf.py

```diff
@@ -26,6 +27,7 @@
 UDF_PYTHON_DEPENDENCIES_ARCHIVE_NAME = "udf-py-deps.zip"
 
 
+@ensure_executor_logging
```

Comment:
@bossie Could ensure_executor_logging be a performance problem here? It also gets called from the driver, so does not need the attribute in every case. But it is handy to just put it on one location

Reply:
(Build currently failing due to error on master)

Reply:
note that `ensure_executor_logging` only does stuff the first time the decorated function is called, after that it's basically a no-op I think